pkg/logs/input/container/kubernetes_scanner.go

```diff
@@ -168,13 +168,43 @@ const kubernetesIntegration = "kubernetes"
 
 // getSource returns a new source for the container in pod
 func (s *KubeScanner) getSource(pod *kubelet.Pod, container kubelet.ContainerStatus) *config.LogSource {
-	return config.NewLogSource(s.getSourceName(pod, container), &config.LogsConfig{
-		Type:    config.FileType,
-		Path:    s.getPath(pod, container),
-		Source:  kubernetesIntegration,
-		Service: kubernetesIntegration,
-		Tags:    s.getTags(container),
-	})
+	var cfg *config.LogsConfig
+	if annotation := s.getAnnotation(pod, container); annotation != "" {
+		cfg, err := config.Parse(annotation)
+		if err != nil {
+			log.Warnf("Invalid pod annotation for pod %v, container %v: %v", pod.Metadata.Name, container.Name, err)
+			return nil
```

Comment:
Do we want to abort creating the source for this pod just because the label is misconfigured ?

Reply:
This is some we agreed on with @NBParis